Support pulling git tokens from ENV vars

accorvin opened the pull request:
If a token value in the config file matches the format
"ENV.{ENV_VAR_NAME}" then the value for the token will be pulled from
the environment variable "ENV_VAR_NAME". This will make it easier to
store a full config file in git and store the tokens in Openshift
secrets.

ddelabru commented:
This looks good, I gave it a test run and it did what it says on the tin. My only quibble is that your description here makes it seem like maybe one should put the environment variable name in curly braces in the review-rot config file, when doing so actually causes review-rot to fail to read the correct environment variable.

accorvin commented:
@ddelabru thanks for the input. After reading the description you're absolutely right that it was unclear. I've updated the commit message and added some comments around the changed code to make it clearer how the new feature works.
